workflow/scripts/filter_krakenuniq.R

```diff
@@ -8,9 +8,9 @@ df <- read.delim(args[1],comment.char="#",header=TRUE)
 
 #Filtering KrakenUniq output with respect to depth (taxReads) and breadth (kmers) of coverage
 colnames(df)[1] <- "Pers_Reads"
-df <- df[df$kmers>1,]
+df <- df[df$kmers>args[3],]
```

Comment:
Is not in R like in python that args need to be converted if they are supposed to be integers? In my testing, `args[3]` and `args[4]` are of type "character", so filtering the way it is now wouldn't work.

I suggest rewriting so that `args` are assigned to variables like so:
`min_kmers <- as.integer(args[3])`

which also makes the code more readable

Reply:
Good point. Renaming all args to variable names makes for improved comprehension of code.